### PR TITLE
Shearlab inv adj new

### PR DESCRIPTION
**Pull request by arsenal9971**
Finally, lets see if it works

### CODE REVIEW

**Comment by adler-j** [comment]
So, before I go ahead here. We can close both #1358 and #1356?

File: odl/contrib/shearlab/shearlab_operator.py [hunk 1]
```diff
@@ -171,8 +179,51 @@ def _call(self, x):
...
+                    odl.contrib.shearlab.ShearlabOperator
+                    """
+
+                    def __init__(self):
+                        """Initialize a new instance.
+                        Parameters
+                        ----------
+                        op : `ShearlabOperator`
+                              The operator which this should be the
+                              inverse of the adjoint of.
```
Comment (kohr-h) [inline odl/contrib/shearlab/shearlab_operator.py]: Remove the Parameters section

**Comment by kohr-h** [comment]
> So, before I go ahead here. We can close both #1358 and #1356?

Okay, since the other two are littered with off-topic discussions about git, let's go with this one. I'll close the other two.

File: odl/contrib/shearlab/shearlab_operator.py (hunk 1 above)
Comment (aringh) [reply odl/contrib/shearlab/shearlab_operator.py]: As pointed out in #1358, this also goes for lines 78, 117, and 164 (although these lines are not changed in these commits).

**Comment by arsenal9971** [comment]
I already did the change, push to my branch `fix_shearlab_invadj`, and then  `git push -f origin kohr-h/shearlab_inv_adj_new:fix_shearlab_invadj.` But I don't see the change in this pull request.

**Comment by kohr-h** [comment]
Then I guess your branch has the update. It doesn't make any more sense to push *my* branch again to `origin/fix_shearlab_invadj` (which you apparently did), because it doesn't have any changes. 

What you instead need to do is `git push origin fix_shearlab_invadj` or more explicitly `git push fix_shearlab_invadj:fix_shearlab_invadj` (*your* local `fix_shearlab_invadj` to the remote one in `origin` with the same name).

**Pull request closed without merging**